content/en/docs/concepts/security/rbac-good-practices.md

```diff
@@ -0,0 +1,172 @@
+Secrets they would not have through RBAC directly.
+
+### Persistent volume creation
+
+As noted in the [PodSecurityPolicy](/docs/concepts/policy/pod-security-policy/#volumes-and-file-systems) documentation, access to create PersistentVolumes can allow for escalation of access to the underlying host.
+
+### Access to `proxy` subresource of Nodes
+
+Users with access to the proxy sub-resource of node objects have rights to the Kubelet API, 
+which allows for command execution on every pod on the node(s) which they have rights to. 
+This access bypasses audit logging and admission control, so care should be taken before 
+granting rights to this resource.
+
+### Escalate right
+
+Generally the RBAC system prevents users from creating clusterroles with more rights than 
+they possess. The exception to this is the `escalate` right. As noted in the [RBAC documentation](/docs/reference/access-authn-authz/rbac/#restrictions-on-role-creation-or-update),
+users with this right can effectively escalate their privileges.
+
+### Bind right
+
+Similar to the `escalate` right, granting users this right allows for bypass of Kubernetes 
+in-built protections against privilege escalation, allowing users to create bindings to 
+roles with rights they do not already have.
+
+### Impersonate right
+
+This right allows users to impersonate and gain the rights of other users in the cluster. 
+Care should be taken when granting it, to ensure that excessive permissions cannot be gained 
+via one of the impersonated accounts.
+
+### CSRs and certificate issuing
+
+The CSR API allows for users with `create` rights to CSRs and `update` rights on `certificatesigningrequests/approval` 
+where the signer is `kubernetes.io/kube-apiserver-client` to create new client certificates 
+which allow users to authenticate to the cluster. Those client certificates can have arbitrary 
+names including duplicates of Kubernetes system components. This will effectively allow for privilege escalation.
+
+### Token request
+
+Users with `create` rights on `serviceaccounts/token` can create TokenRequests to issue 
+tokens for existing service accounts. 
+
+### Control admission webhooks
+
+Users with control over `validatingwebhookconfigurations` or `mutatingwebhookconfigurations` 
+can control webhooks that can read any object admitted to the cluster, and in the case of 
+mutating webhooks, also mutate admitted objects.
+
+
+## Kubernetes RBAC - denial of service risks
+
+### Object creation denial-of-service {#object-creation-dos}
+Users who have rights to create objects in a cluster may be able to create sufficient large 
+objects to create a denial of service condition, as discussed in
+[etcd used by Kubernetes is vulnerable to OOM attack](https://github.com/kubernetes/kubernetes/issues/107325). This may be
+specifically relevant in multi-tenant clusters if semi-trusted or untrusted users 
+are allowed limited access to a system.
```

Comment:
object size and object quantity are two dimensions that can trigger this. referencing resourcequota docs as a mitigation for object quantity could be useful

Reply:
updated this section .